samples/net/zperf/prj.conf

```diff
@@ -40,3 +40,4 @@ CONFIG_LOG=y
 CONFIG_SHELL_CMDS_RESIZE=n
 
 CONFIG_CACHE_MANAGEMENT=y
+CONFIG_SPEED_OPTIMIZATIONS=y
```

Comment:
If IPv6 is used, it would be useful to enable `CONFIG_NET_TCP_IPV6_ND_REACHABILITY_HINT=y`